bundles/org.eclipse.osgi/supplement/src/org/eclipse/osgi/util/NLS.java

```diff
@@ -125,33 +125,38 @@ public static String bind(String message, Object[] bindings) {
 	}
 
 	/**
-	 * Initialize the given class with the values from the message properties specified by the
-	 * base name.  The base name specifies a fully qualified base name to a message properties file,
-	 * including the package where the message properties file is located.  The class loader of the
-	 * specified class will be used to load the message properties resources.
+	 * Initialize the given class with the values from the message properties
+	 * specified by the base name. The base name specifies a fully qualified base
+	 * name to a message properties file, including the package where the message
+	 * properties file is located. The class loader of the specified class will be
+	 * used to load the message properties resources.
 	 * <p>
-	 * For example, if the locale is set to en_US and <code>org.eclipse.example.nls.messages</code>
-	 * is used as the base name then the following resources will be searched using the class
-	 * loader of the specified class:
+	 * For example, if the locale is set to en_US and
+	 * <code>org.eclipse.example.nls.messages</code> is used as the base name then
+	 * the following resources will be searched using the class loader of the
+	 * specified class:
 	 * </p>
+	 * 
 	 * <pre>
 	 *   org/eclipse/example/nls/messages_en_US.properties
 	 *   org/eclipse/example/nls/messages_en.properties
 	 *   org/eclipse/example/nls/messages.properties
 	 * </pre>
 	 * <p>
-	 * The properties files are read using the default encoding for resource bundles:
+	 * The properties files are read using the default encoding for resource
+	 * bundles:
+	 * </p>
```

Comment:
Formatting changed quite a bit, but this got moved here from after the the `</ul> below.